Makefile

```diff
@@ -4,10 +4,11 @@ VERSION := 0.5.4
 SERVICE_NAME := redis-operator
 
 # Docker image name for this project
-IMAGE_NAME := spotahome/$(SERVICE_NAME)
+IMAGE_NAME ?= spotahome
```

Comment:
I'd name this `ORGANIZATION` to be more consistent to it's usage.

Reply:
Actually this variable will be used for the image name to be used, in this case the image name happen to be named as `spotahome`.

Reply:
I see that afterwards, this will be used this way:
```
REPOSITORY := $(REPO_SERVER)/$(IMAGE_NAME)/$(SERVICE_NAME)
```
For me, it has more sense to have the following:
```
SERVICE_NAME := redis-operator
ORGANIZATION ?= spotahome
REPO_NAME ?= quay.io
IMAGE := $(REPO_NAME)/$(ORGANIZATION)/$(SERVICE_NAME)
```

Reply:
After reading [docker documentation](https://docs.docker.com/engine/reference/commandline/images/), I'd use the following that is a mix between both:

```
SERVICE_NAME := redis-operator
ORGANIZATION ?= spotahome
REPO_SERVER ?= quay.io
REPOSITORY := $(REPO_SERVER)/$(ORGANIZATION)/$(SERVICE_NAME)
```

Any thoughts? Do you agree with this?